src/main/drivers/adc_stm32f10x.c

```diff
@@ -67,7 +67,6 @@ const adcTagMap_t adcTagMap[] = {
 
 void adcInit(const adcConfig_t *config)
 {
-
```

Comment:
Good point.

So far we have always 'dragged along' F1 and F3, not adding new features and not done much improvement, but applied low level changes like this to F1 and F3.

This was done on the background that users would be able to still build custom builds of current versions of Betaflight for F1 or F3 if they remuved enough features they don't need.
But obviously we don't know how much this is used, or, since none of these builds are done in our CICD setup, if this actually still works.

So I think we should be discussing this, but probably better outside of this pull request.